assets/prototype/domain/shared/predicates/dateUtilities/add-subtract.ts

```diff
@@ -0,0 +1,53 @@
+import {
```

Comment:
Instead of creating 2 files for both `add` and `subtract`, I've decided to take the same approach as in `moment.js`.
The reason being that there are some duplication.
More info: https://github.com/moment/moment/blob/develop/src/lib/moment/add-subtract.js